docs/site/migration/auth/example.md

```diff
@@ -446,143 +448,64 @@ Try the 'team-member' role:
-? Is it required?: Yes
-? Default value [leave blank for none]:
-
-Lets add another property to Team
-Enter an empty property name when done
-
-? Enter the property name:
-   create src/models/team.model.ts
-   update src/models/index.ts
-
-Model Team was created in src/models/
+lb4 import-lb3-models <path_to_loopback-example-access-control> --outDir src/models
 ```
 
-Create Project Model
+Choose model `User`, `Project`, `Team` for the prompt.
 
-```sh
-$ lb4 model
-? Model class name: Project
-? Please select the model base class Entity (A persisted model with an ID)
-? Allow additional (free-form) properties? Yes
-Model Project1 will be created in src/models/project.model.ts
+Considering the difference between the original application and the migrated
+one, you need to adjust the properties and add the relations.
 
-Lets add a property to Project
-Enter an empty property name when done
+- For `Project`, open file `src/models/project.model.ts`
 
-? Enter the property name: id
-? Property type: number
-? Is id the ID property? Yes
-? Is id generated automatically? No
-? Is it required?: Yes
-? Default value [leave blank for none]:
+  - decorate `ownerId` with `@belongsTo(() => User)` and remove the generated
+    `@property` decorator.
+  - `balance` should be a required property: change it from `balance?` to
+    `balance`
 
-Lets add another property to Project
-Enter an empty property name when done
+- For `Team`, open file `src/models/team.model.ts`
 
-? Enter the property name: name
-? Property type: string
-? Is it required?: Yes
-? Default value [leave blank for none]:
+  - replace `memberId` with `memberIds` as an array:
+    ```ts
+      @property({
+        type: 'array',
+        itemType: 'number',
+        required: true,
+      })
+      memberIds: number[];
+    ```
 
-Lets add another property to Project
-Enter an empty property name when done
+- For `User`, open file `src/models/user.model.ts`
 
-? Enter the property name: balance
-? Property type: number
-? Is it required?: Yes
-? Default value [leave blank for none]:
+  - remove `password` because we have a `UserCredential` model created to
+    separate it from `User`
+  - add relations:
 
-Lets add another property to Project
-Enter an empty property name when done
+  ```ts
+    @hasMany(() => Team, {keyTo: 'ownerId'})
+    teams: Team[];
 
-? Enter the property name:
-   create src/models/project.model.ts
-   update src/models/index.ts
+    @hasOne(() => UserCredentials)
+    userCredentials: UserCredentials;
+  ```
 
-Model Project was created in src/models/
-```
+- For all the three models above, allow generating their `id` field:
+  ```ts
+    @property({
+    type: 'number',
+    id: 1,
+    // change it from `true` to `false`
+    generated: false,
+    updateOnly: true,
+  })
+  // change it from optional `id?` to required `id`
+  id: number;
+  ```
 
-Create UserCredentials
+For model `UserCredentials`, you need to create it by `lb4 model`:
```

Comment:
I'd suggest to move the model creation right after importing other models, so that we'll have all the models created by that time. Then the next few steps are about changing the model.ts and adding relations.